Add info on use of CPNS deaths vs ONS deaths (#617)
opensafely/documentation: docs/study-def-variables.md

## CPNS

These variables are derived from the COVID-19 Patient Notification System (CPNS), which collects info on all in-hospital covid-related deaths.
For more information, see the [CPNS data section](dataset-cpns.md). 

!!! note

>     Please note that CPNS is restricted to in-hospital covid-related deaths, for covid-related deaths in any setting, ONS-registered deaths where cause of death       matches [COVID-19 coding in ICD-10](https://www.who.int/classifications/icd/COVID-19-coding-icd10.pdf) is generally more useful.

Reviewer: nit:
```suggestion
    CPNS is restricted to in-hospital covid-related deaths only. For covid-related deaths in any setting, ONS-registered deaths where cause of death matches [COVID-19 coding in ICD-10](https://www.who.int/classifications/icd/COVID-19-coding-icd10.pdf) is generally more useful.
```

Perhaps very slightly more readable :arrow_up: ?

Reviewer 2: I'd agree 🙂 I like the "Commit suggestion" button, too.
